js/data.js

```diff
@@ -750,6 +860,28 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze"],
         nameLV: "Non-Uniform Memory Access (NUMA) - Herausforderungen paralleler Systeme "
     },
+    openHPI: {
```

Comment:
I guess there will be many openHPI seminars in the future, so maybe call these 'openHPI1' or 'openHPIVirtualization'.